README.md

```diff
@@ -7,7 +7,19 @@ will filter those out to show Keto recipies with a filter for Vegan and Gluten F
 ### Project Team
 Nicolai Safai, Phyllis wong, and Leslie Kimm
 
-### Roles - TBD
-Phyllis Wong
-Nicolai Safai
-Leslie Kimm
+### Roles
+**Lead Backend Engineer / PM**<br>
+Nicolai Safai<br><br>
+**Lead Frontend Engineer**<br> 
+Phyllis Wong<br><br>
+**Senior Engineer / QA**<br>
+Leslie Kimm<br><br>
+
+### Milestones
+- [x] **Sep 28** Shipped to Heroku
```

Comment:
👍 reminder that i need to point our domain, aadding that to my list of things to do today!

Reply:
Awesome, I have a really good resource I'll send via slack.